sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/RateStreamProviderSuite.scala

```diff
@@ -308,30 +308,17 @@ class RateStreamProviderSuite extends StreamTest {
       "rate source does not support user-specified schema"))
   }
 
-  test("continuous in registry") {
```

Comment:
we don't need this test now. With the new `TableProvider` abstraction, the lookup logic is unified between microbatch and continuous